### ISSUE TITLE
No LGPL-2.0

### ISSUE THREAD

**Issue by Wallby** [github/choosealicense.com#946]
E.g. [gtk 3 is licensed w. the library GPL](https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/COPYING) so it seems relevant.

**Comment by mlinksva** [comment]
Yes https://choosealicense.com/licenses/lgpl-3.0/

**Comment by mlinksva** [comment]
Oh I see you mean LGPL-2.0 prior to being renamed "Lesser"; fine with me to add, feel free to submit a PR.

**Comment by Pukimaa** [comment]
The official name is "GNU Library General Public License v2 **or later**". So I guess the rules of LGPL3 and 2 are the same?

**Comment by Pacerier** [comment]
@Pukimaa, yes "_the rules_" can mean anything, but no the text is different.

**Comment by Pacerier** [comment]
@pukimaa re "*official*" [for `LGPL-2.0-or-later`]; that's the official spdx name. The official name is "[GNU Lesser General Public License](https://archive.is/eq5rv#selection-625.19-629.149)" though it was "[GNU Library General Public License](https://archive.is/btlF9#selection-913.97-913.131)".

**Comment by Tarekchehahde** [comment]
Opened https://github.com/github/choosealicense.com/pull/1359 — adds hidden LGPL-2.0 page, SPDX text, using examples, and links from LGPL-2.1.
